src/Umbraco.Web.UI.Client/src/views/components/umb-groups-builder.html

```diff
@@ -159,6 +160,7 @@
                             sortable="sortingMode"
                             on-edit="editPropertyTypeSettings(property)"
                             on-remove="deleteProperty(group.properties, property)"
+                            on-click-composition="openDocumentType(documentTypeId)"
```

Comment:
Maybe content type instead here? Maybe someday we can use composition on media types and member types as well?

Reply:
Same as the comment above here. And compositions are already possible on media types right?

Reply:
Ahh, yes you're right 👍 
Then it would make sense to use "content type" naming instead.

Reply:
Will update it later tonight (along with the other places where it is used)

Reply:
Renamed the functions